test/package/cjs/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "clean": "rimraf dist && rimraf node_modules",
+    "clean": "rm -rf dist && rm -rf node_modules && rm -rf target && rm -f yarn*",
```

Comment:
the use of `rm` may not be safe for Windows users, not sure if that is a concern here

Reply:
not much of a concern here. but we need to be able to wipe everything before node_modules installed. so using an NPM module to wipe stuff doesn't make sense.